lms/templates/ccx/schedule.underscore

```diff
@@ -26,7 +26,7 @@
           <tr class="sequential collapsed" data-depth="2"
               data-location="<%= chapter.location %> <%= child.location %>">
             <td class="unit">
-              <a href="#"><i class="fa fa-caret-right toggle-collapse"></i></a>
+              <a href="#" class="toggle-collapse"><i class="fa fa-caret-right"></i></a>
```

Comment:
Needs rebase. Done this in #9114 
https://github.com/edx/edx-platform/pull/9114/files#diff-f15570680848e2d23dbfad465e79372dR26